Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM opensciencegrid/osg-wn
```

Comment:
I do want this image to eventually be based off of `software-base` and supervisord on some level, i.e. this is good if we base `osg-wn` on `software-base`

@matyasselmeci any thoughts on whether or not we should base `osg-wn` on `software-base`?

Reply:
It would be really hard to base osg-wn on software-base because osg-wn contains a huge set of tags built basically on the cartesian product of `{3.3,3.4,3.5} * {devel,testing,release} * {el6,el7,el8}` (minus whichever of those combinations are impossible) whereas software-base is only a small subset of those. (Although I wouldn't mind reducing osg-wn to a small subset of those as well...)

Also supervisord is for daemons -- not sure what daemons would be in the wn-client except for maybe cron to run a cron job to update CA certs and VO data or something.

Reply:
Boy, we certainly should get rid of those 3.3/3.4/el6 tags.

> Also supervisord is for daemons -- not sure what daemons would be in the wn-client except for maybe cron to run a cron job to update CA certs and VO data or something.

We could certainly override the supervisord command but that does feel a little silly I guess. Instead, I guess we should base the backfill container off of `software-base` and install the appropriate WN packages.